pyproject.toml

```diff
@@ -56,13 +56,13 @@ keywords = [
 python = ">=3.8,<3.11"
 Django = ">=2.2,<=4.0"
 django-crispy-forms = "1.13.0"
-wagtail = ">=2.15"
 
 [tool.poetry.dev-dependencies]
 pre-commit = "2.15.0"
 beautifulsoup4 = ">=4.8,<4.10.0"
 coverage = "6.1.1"
 pymdown-extensions = "9.0"
+poetry = "1.1.13"
```

Comment:
I find this surprising – have never seen a Poetry project depending on Poetry itself. Isn’t this redundant with the `build-system` section at the end of this file?

Reply:
Hmm I do have Poetry installed globally for the projects I use it on. I don’t know any other way to use it, but I’m by no means a regular Poetry user.